pkg/inventory/inventory-client_test.go

```diff
@@ -104,15 +105,13 @@ func TestGetClusterInventoryInfo(t *testing.T) {
 			if !tc.isError && err != nil {
 				t.Fatalf("unexpected error received: %s", err)
 			}
-			if clusterInv != nil {
-				wrapped := WrapInventoryObj(clusterInv)
-				clusterObjs, err := wrapped.Load()
-				if err != nil {
-					t.Fatalf("unexpected error received: %s", err)
-				}
-				if !tc.localObjs.Equal(clusterObjs) {
-					t.Fatalf("expected cluster objs (%v), got (%v)", tc.localObjs, clusterObjs)
-				}
+			require.NotNil(t, clusterInv)
+			wrapped := WrapInventoryObj(clusterInv)
+			require.NoError(t, err)
```

Comment:
Don't need this here since the previous line doesn't produce an error

Reply:
Ah missed that because the next PR has it return an error. Removed